style.css

```diff
@@ -737,16 +805,548 @@ ul {
+    flex-direction: row;
+    padding: 24px;
+    gap: 46px;
+    background: #fff;
+    border: 1px solid #dfe1e6;
+    margin-top: 100px;
+    height: 60vh;
+    width: 100%;
+    border-radius: 24px;
+  }
+
+  .card-container img {
+    width: 50%;
+    height: 100%;
+    border-radius: 8px;
+  }
+
+  .faha2 {
+    order: 2;
+  }
+
+  .work-fonony {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    padding-left: 2%;
+    margin-left: 10%;
+    width: 50%;
+    margin-top: 5rem;
+  }
+
+  .card-tittle {
+    margin-top: -15rem;
+    margin-left: -5.5rem;
+  }
+
+  .frame2-img {
+    margin-top: -1rem;
+    margin-left: -5.7rem;
+  }
+
+  .frame3-img {
+    display: flex;
+    align-items: center;
+  }
+
+  .project-info {
+    list-style: none;
+    display: flex;
+    justify-content: space-between;
+    padding: 0;
+    margin: 0;
+    width: 100%;
+    font-size: 16px;
+    margin-left: -5.5rem;
+    margin-top: -3rem;
+    margin-bottom: 1rem;
+    font-weight: bold;
+    flex-direction: row;
+    align-items: center;
+    gap: 8px;
+    order: 2;
+  }
+
+  ul.project-info li {
+    list-style-type: none;
+    position: relative;
+    padding-right: 5px;
+    white-space: nowrap;
+  }
+
+  ul.project-info li:not(:first-child)::before {
+    content: "\2022";
+    font-size: 40px;
+    position: absolute;
+    left: -3rem;
+    top: 0;
+  }
+
+  .fb {
+    font-size: 18px;
+    font-weight: 500;
+    line-height: 24px;
+    text-align: left;
+    color: #3b5998;
+  }
+
+  .full {
+    font-size: 18px;
+    font-weight: 400;
+    line-height: 24px;
+    text-align: left;
+    color: #6b778c;
+  }
+
+  .year {
+    font-size: 18px;
+    font-weight: 400;
+    line-height: 24px;
+    text-align: left;
+    color: #6b778c;
+  }
+
+  #fb-title {
+    margin-top: -15rem;
+    margin-left: -5.5rem;
+  }
+
+  #fb-info {
+    list-style: none;
+    display: flex;
+    justify-content: space-between;
+    padding: 0;
+    margin: 0;
+    width: 100%;
+    font-size: 16px;
+    margin-left: -5.5rem;
+    margin-top: -3rem;
+    margin-bottom: 1rem;
+    font-weight: bold;
+    flex-direction: row;
+    align-items: center;
+    gap: 8px;
+    order: 2;
+  }
+
+  #back-list {
+    margin-left: -5.3rem;
+    margin-top: -10.5rem;
+  }
+
+  #fd-buttons {
+    margin-left: -5.4rem;
+    margin-top: 5rem;
+  }
+
+  #description-fb {
+    margin-top: -2rem;
+    width: 100%;
+    margin-left: -5.3rem;
+  }
+
+  .description {
+    margin-top: -2rem;
+    width: 100%;
+    margin-left: -5.3rem;
+  }
+
+  ul.languages-list li {
+    white-space: nowrap;
+  }
+
+  .languages-list {
+    margin-left: -5.3rem;
+    margin-top: -8.5rem;
+  }
+
+  .buttons {
+    margin-left: -5.4rem;
+    margin-top: 2rem;
+    background: #4053fc;
+    color: #fff;
+  }
+
+  /* About section */
+
+  .about-me {
+    display: grid;
+    flex-direction: row;
+    width: 100%;
+    height: 90vh;
+    padding: 56px 14px 0 16px;
+  }
```

Comment:
- [ ] Good job implementing the changes on the bout me section suggested by the previous reviewer. :smile: But it seems like the changes you made affected other parts of your code, On the screen size between `765px` and `1365px` the technology section seems to overflow the screen, Please look into this. :+1: :smile: 
- Screenshot for reference:
![Screenshot from 2023-04-08 10-58-54](https://user-images.githubusercontent.com/107632780/230716025-5f7d8eb7-5583-4c01-9f31-f9ed6f1d3c30.png)